src/config/database.js

```diff
@@ -0,0 +1,19 @@
+import mysql from 'mysql';
+import { ConfigService } from './ConfigService.js';
+
+const configService = new ConfigService();
+
+const connection = mysql.createPool({
```

Comment:
```suggestion
const connection = mysql.createPool({
```
class Database {
  getDb() {
    const dbConfig = {}
    return mysql.createPool(dbConfig);
  }
}